src/commands/dev/dev.js

```diff
@@ -593,6 +611,7 @@ const createDevCommand = (program) => {
         .choices(['cache', 'mock', 'update'])
         .default('cache'),
     )
+    .addOption(new Option('--country <geoCountry>', 'Two letter country code').argParser(validateGeoCountryCode))
```

Comment:
I think this gives a bit more information about what the flag is for.

```suggestion
    .addOption(new Option('--country <geoCountry>', 'Two-letter country code to use as mock geolocation (to be used with --mock)').argParser(validateGeoCountryCode))
```